src/react/sebs_snacks/src/index.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { render } from 'react-dom';
+import {
+  BrowserRouter,
+  Routes,
+  Route
+  } from 'react-router-dom'
+
+import App from './App';
+import AboutmePage from './components/aboutme';
+import RecipePage from './components/recipePage';
+
+const rootElement = document.getElementById("root");
+
+render(
+  <BrowserRouter>
+    <Routes>
+      <Route path="/" element={<App />} />
+      <Route path="/aboutme" element={<AboutmePage />} />
+      <Route path="/recipe" element={<RecipePage />}>
+        <Route path=":recipeID" element={<RecipePage />} />
+      </Route>
+      <Route path="*" element={<App />} /> {/*In case the path does not catch anything */}
```

Comment:
You can do <Route exact path="/" element={<App />}>  on line 18 and get rid of this. Adding exact basically adds a catch all